packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -727,94 +727,88 @@ export const ForumEventPoll = ({
                     className={classNames(
                       classes.tick,
                       isDragging.current && classes.tickDragging,
-                      tickIndex === Math.floor(NUM_TICKS / 2) &&
-                        classes.centralTick
+                      tickIndex === Math.floor(NUM_TICKS / 2) && classes.centralTick
                     )}
                   />
                 ))}
                 {/* User Vote */}
-                <div
-                  className={classNames(
-                    classes.userVote,
-                    classes.currentUserVote,
-                    isDragging.current && classes.currentUserVoteDragging,
-                    hasVoted && classes.currentUserVoteActive
-                  )}
-                  onPointerDown={startDragVote}
-                  style={{
-                    left: `${votePos}%`,
-                    transform: "translateX(-50%)",
-                  }}
+                <AnalyticsContext
+                  pageElementContext="forumEventUserIcon"
+                  forumEventId={event?._id}
```

Comment:
I would look at this side by side, a lot of this is due to adding an analytics context